api/src/main/java/org/pmiops/workbench/actionaudit/auditors/UserServiceAuditor.java

```diff
@@ -15,7 +15,10 @@ void fireUpdateDataAccessAction(
       Agent agent);
 
   void fireAdministrativeBypassTime(
-      long userId, BypassTimeTargetProperty bypassTimeTargetProperty, Optional<Instant> bypassTime);
+      long userId,
+      BypassTimeTargetProperty bypassTimeTargetProperty,
+      Optional<Instant> bypassTime,
```

Comment:
Can you please make sure the order matches other auditor functions? I believe it's usually previous then new. IJ has a Change Signature refactoring for that. Also please rename bypassTime to newBypassTime to make it foolproof.

Reply:
ok